client/src/hooks/store/useLeaderBoardStore.tsx

```diff
@@ -75,4 +76,16 @@ export const useSubscriptionToHyperstructureEvents = () => {
 
     return () => subscription.unsubscribe();
   }, []);
+
+  const gameEndedEntityId = useEntityQuery([Has(GameEnded)]);
```

Comment:
why not use useComponentValue directly ? we already know the entity id of the GameEnded entity

Reply:
oh didnt realize winner_address was a key (which isnt needed imo) but let's keep it like this then, lgtm